### DISCUSSION TITLE
New version 2.0.5!

### DISCUSSION THREAD

**Discussion by rzeldent** [rzeldent/esp32-smartdisplay#127] [Announcements]
More parametrization
Use of PSRAM so amazing fast!
Fixes

**Comment by Toekie** [comment]
Just wanted to thank you for this amazing work. Im a novice and was struggle with the nextion boards for a while but since these have gotten way more expensive the last years i wanted to try these esp screens but couldnt find proper libraries. These are great!

**Reply by rzeldent** [reply]
Thanks for mentioning this! You're welcome... Enjoy!
